spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubEmulatorConfigurationTests.java

```diff
@@ -43,8 +43,6 @@
 	private ApplicationContextRunner contextRunner = new ApplicationContextRunner()
 			.withPropertyValues("spring.cloud.gcp.pubsub.emulator-host=localhost:8085",
 					"spring.cloud.gcp.projectId=test-project",
-					"spring.cloud.gcp.pubsub.trusted-packages[0]=a",
```

Comment:
For all the exposing-config related work we've pushed for recently, it feels like a big step in the opposite direction to take away this trusted packages config functionality. 

From reading it seems like JSON Jackson deserialization is never to be used for serious business without trusted packages settings, and taking this away really feels like we're leaving the user out in the cold providing and configuring his or her own objectmapper.

Reply:
There so many ways to do serialization that I don't think we need to over-prescribe it. Even on the surface the property doesn't make sense to be part `spring.cloud.gcp.pubsub.*` because it has nothing to do with Pub/Sub. Furthermore, the Jackson documentation encourages the use of the annotations instead, like I did in the `AllowedPayload`.
Lastly, this was Artem's suggestion to introduce `trusted-packages` and he doesn't seem to mind this change.

Reply:
Right, I'm fine to dropping this one, since it definitely is out of this project scope.

Or annotations way or provided `ObjectMapper` instance configuration.
Since we have came to the situation when we would like do not depend here on Spring Integration, we don't have choice unless just document it with an appropriate sample, e.g. pointing to that Spring Integration's `JacksonJsonUtils` and so on.

We have many other similar approaches in Spring Kafka and Spring AMQP (maybe others) and I don't see that we expose there a `trusted packages` property. So, we are going to be good here to live without that as well.

Sorry for confusing in the begging: this project is not my full-time responsibility to think about all the features properly 😢